### DISCUSSION TITLE
OMR-Bypass no work well.

### DISCUSSION THREAD

**Discussion by Melania95** [Ysurac/openmptcprouter#3129] [General]
Hi friends, could someone explain to me how OMR-Bypass works? I have 3 4G connections and everything works perfectly, but I have some inaccessible sites. If I set https://www.reddit.com as the domain to bypass and set "eth1" as the interface, the site is ok (both by setting the OMR DNS on the PC and without them), but I lose the aggregation because I do a speedtest I no longer connect to the IP of the VPS but to the IP of one of the providers! Could it be that I made some settings wrong or is all of this normal? However, if I select the PC's Mac Address, performance becomes much worse. I believe that the OMR-Bypass should only use the selected connection to access that specified site, maintaining the IP of the VPS and therefore the aggregation for everything else!! Or not? Thank you all.
Edit: I use 0.59.1 kernel 5.4

**Comment by Ysurac** [comment]
When you bypass a domain/ip, the connection is direct from one of your connection. So yes you loose aggregation to this domain/ip only.
You can loose connection to the VPS if the VPS IP is in the bypass list or if DNS return strange results.
You can check bypassed IP in ```ipset list```

**Reply by Melania95** [reply]
Thank you so much....I have verified `ipset list` with SSH and i have found my Vps Ip here, i think is this the error:
`Name: ssr_def_dst_bypass_
Type: hash:net
Revision: 6
Header: family inet hashsize 64 maxelem 65536
Size in memory: 1536
References: 2
Number of entries: 20
Members:
240.0.0.0/4
198.18.0.0/15
192.175.48.0/24
xxx.xxx.xxx.xxx
`
What should I do? Thanks...

**Reply by Ysurac** [reply]
This one is normal.
Status page is all green ? What is the result of ```uci show omr-bypass``` ?

**Reply by Melania95** [reply]
Yes bonding work well status page is all green...this is my `uci show omr-bypass`:

```
root@OpenMPTCProuter:~# uci show omr-bypass
omr-bypass.all=interface
omr-bypass.m6replay=proto
omr-bypass.m6replay.url='m6web.fr' '6play.fr' '6cloud.fr'
omr-bypass.mycanal=proto
omr-bypass.mycanal.url='mycanal.fr' 'canal-plus.com' 'canalplus.com' 'canalplus-                                                                                                                                                             cdn.net' 'canalplus.pro' 'canal-plus.net'
omr-bypass.minecraft=proto
omr-bypass.minecraft.url='authserver.mojang.com'
omr-bypass.lesnumeriques=proto
omr-bypass.lesnumeriques.url='lesnumeriques.com' 'botscorner.com' 'app.botscorne                                                                                                                                                             r.com'
omr-bypass.disneyplus=proto
omr-bypass.disneyplus.url='bamgrid.com' 'disney-plus.net'
omr-bypass.amazonvideo=proto
omr-bypass.amazonvideo.url='cloudfront.net' 'llnw.net'
omr-bypass.lo=interface
omr-bypass.lo.id='7'
omr-bypass.eth0=interface
omr-bypass.eth0.id='8'
omr-bypass.eth1=interface
omr-bypass.eth1.id='3'
omr-bypass.eth2=interface
omr-bypass.eth2.id='4'
omr-bypass.eth3=interface
omr-bypass.eth3.id='5'
omr-bypass.eth4=interface
omr-bypass.eth4.id='10'
omr-bypass.tun0=interface
omr-bypass.tun0.id='1200'
omr-bypass.Bridge=interface
omr-bypass.Bridge.id='8'
omr-bypass.@dpis[0]=dpis
omr-bypass.@dpis[0].proto='whatsapp'
omr-bypass.@dpis[0].interface='eth2'
omr-bypass.@dpis[1]=dpis
omr-bypass.@dpis[1].proto='whatsappcall'
omr-bypass.@dpis[1].interface='eth2'
omr-bypass.@dpis[2]=dpis
omr-bypass.@dpis[2].proto='whatsappfiles'
omr-bypass.@dpis[2].interface='eth2'
omr-bypass.@dpis[3]=dpis
omr-bypass.@dpis[3].proto='telegram'
omr-bypass.@dpis[3].interface='eth2'
omr-bypass.@dpis[4]=dpis
omr-bypass.@dpis[4].proto='reddit'
omr-bypass.@dpis[4].interface='eth2'
omr-bypass.@domains[0]=domains
omr-bypass.@domains[0].name='https://www.tgcom24.mediaset.it'
omr-bypass.@domains[0].interface='eth2'
```

**Reply by Melania95** [reply]
I'm using OMR on Proxmox...can be this the cause?

**Reply by Ysurac** [reply]
As domain name you should only set domain, without the https:// or http://
You can also set the top domain, for example google.fr for www.google.fr, test.google.fr,... all subdomains will also be bypassed.

**Reply by Melania95** [reply]
Already tried...it doesn't work anyway
